app/views/layouts/application.html.erb

```diff
@@ -122,11 +122,14 @@
         items: [
           {
             href: "https://support.publishing.service.gov.uk/technical_fault_report/new",
-            text: "Raise a support request"
+            text: "Raise a support request",
+            attributes: {target: "_blank"}
```

Comment:
The linter doesn't seem to cover erb, but if it did it'd be asking you to put spaces inside the hash

Reply:
Done. Spaces put in